Tests/NIOKitTests/FutureExtensionsTests.swift

```diff
@@ -0,0 +1,39 @@
+import XCTest
+import NIO
+@testable import NIOKit
+
+final class FutureExtensionsTests: XCTestCase {
```

Comment:
This should be conformed to `NIOTestCase`. Then lines 6 - 19 can be removed.

Reply:
Good catch! :)